decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -165,9 +166,10 @@ def profiling_gems
 
         copy_file "bullet_initializer.rb", "config/initializers/bullet.rb"
         copy_file "rack_profiler_initializer.rb", "config/initializers/rack_profiler.rb"
-        Bundler.with_unbundled_env do
-          run "bundle install"
-        end
+      end
+
+      def bundle_install
```

Comment:
Just to be clear I understand what's happening here:
1. You removed the `Bundler.with_unbundled_env { ... }` block to make it possible to pass the bundler env variables if needed?
2. We also need to run `bundle install` when `options[:skip_gemfile]` or `options[:skip_bundle]` is set to false. So this means we are not respecting these configurations, we are just using the `skip_bundle` option to tell "upstream" that we don't want them to run `bundle install` and run it ourselves instead.

Could you clarify these so that I am perfectly following what's happening with these changes?

Reply:
Yes, we need the environment variables now. I introduced `with_unbundled_env` in #8990 otherwise tests weren't passing due the special setup generated in them (creating and importing the gems locally) and the need to test custom ENV variables. But then I realized that this only happened if we call `bundle install` from within the AppGenerator. Not sure why to be honest, but this is the way it used to work.

The --skip-bundle option is not passed to the InstallGenerator so, in principle, it should be respected, *if we want to*. The problem here is that the webpacker commands do not work without a `bundle install`, for instance this one ` rails "webpacker:binstubs"`. The result is that, if we respect this option, the generated Decidim application will be incomplete.

But I think its been broken since the introduction of webpacker. Maybe we can tackle this issue in another PR though. Or maybe we could add some clause to respect the --skip-bundle, skip commands that depend on it and throw a message in the console with a warning that you need to run them in order to complete the installation. What you think?

Reply:
My point too would be to merge this as it is, then I can create an issue with the problem and we can take it in the future,probably when some other improvements need to be done in the generator.

If not, I'm fine with introducing some warnings and skip commands if the option is present. Let's wait for Andres opinion.

Reply:
👍🏽 to going with what works for the majority. As far as I understand, you could always run the changes that you want afterwards if you need that.